tutorials/katacoda/getting-started/setup.md

```diff
@@ -0,0 +1,56 @@
+## Before you begin
+
+Start an experimental Kubernetes cluster in Katacoda with command
+`launch.sh`{{execute}}.
+
+Run `kubectl cluster-info`{{execute}} in the terminal to check its status.
+You should see the following outputs:
+
+```
+Kubernetes master is running at ...
+KubeDNS is running at ...
+
+...
+```
+
+## Installing Tekton
+
+To add Tekton to this experimental Kubernetes cluster, execute the command
+below:
+
+`kubectl apply --filename https://storage.googleapis.com/tekton-releases/latest/release.yaml`{{execute}}
```

Comment:
If we truly want the latest we need to update the URL as this old structure is no longer used and points at Pipelines 0.10.

Updated URL (currently pointing to 0.13.2) is:

```suggestion
`kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml`{{execute}}
```